EsportsHelper/Rewards.py

```diff
@@ -188,10 +193,9 @@ def notifyDrops(self, poweredByImg, productImg, eventTitle, unlockedDate, dropIt
                     }
                     embed = {
                         "title": "Drop!",
-                        "description": f"{self.config.username}",
                         "image": {"url": f"{productImg}"},
                         "thumbnail": {"url": f"{dropItemImg}"},
-                        "fields": [field1, field2, fieldNone, field4, fieldNone, fieldNone],
+                        "fields": [field0, field1, field2, fieldNone, field4, fieldNone, fieldNone],
```

Comment:
should be :
"fields": [field0, field1, fieldNone, field2, field4, fieldNone, fieldNone],